config/locales/ar.yml

```diff
@@ -590,7 +590,7 @@ ar:
   fatality_notice:
     alt_text: شعار وزارة الدفاع
     field_of_operation: ساحة العمليات
-    none_added:
+    none_added: لم تتم إضافة أي إشعارات حول الوفيات في ساحة العمليات هذه حتى الآن.
```

Comment:
Could you reword the commit message a bit here to make a bit more sense? I know what it's getting at now I look at the contents of this commit, but was confused on initial reading.

Reply:
have updated :)